src/gui/tray/Window.qml

```diff
@@ -579,296 +579,11 @@ Window {
 
             model: activityModel
 
-            delegate: RowLayout {
-                id: activityItem
-
-                readonly property variant links: model.links
-
-                readonly property int itemIndex: model.index
-
+            delegate: ActivityItem {  
                 width: parent.width

```

Comment:
use `width: activityListView.width` to avoid warnings when parent is null (happen during creation of delegate instances)